modules/bim/app/seeders/bim/demo_data/bcf_xml_seeder.rb

```diff
@@ -41,7 +41,7 @@ def seed_data!
         filename = project_data_for(key, 'bcf_xml_file')
         return unless filename.present?
 
-        user = User.admin.first
+        user = User.admin.active.first
```

Comment:
User.admin.first now is the system user since 11.3., this is probably not the user you want to import under?

Reply:
Correct. I want the freshly seeded admin user.